tools/ormolu.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# TODO: change to stack project root directory.
+
+
+if [ "`git status -s src/ test/ | grep -v \?\?`" != "" ]; then
+    echo "working copy not clean."
+    if [ "$1" == "-f" ]; then
+        echo "running with -f.  THIS WILL DESTROY YOUR UNCOMMITTED CHANGES."
+        git reset --hard HEAD
```

Comment:
Do we really need to reset to HEAD? If I really want to run this on my uncommitted changes via `-f`, I would expect them to get formatted (which is kind of unsafe, since I don't have a versioned copy of them, but usually fine), but not just dropped completely. Not sure what the advantage of that is.

Reply:
fair enough, i changed that back.